astropy/tests/figures/py310-test-image-mpldev-cov.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Sorry I don't get what you're suggesting here

Reply:
Update the Baseline Image
Open a terminal on system.

Navigate to the directory containing the test results:                  
`cd /home/circleci/project/results/astropy.visualization.wcsaxes.tests.test_frame.TestFrame.test_custom_frame/
`
Rerun the tests to verify everything works:
`pytest --pyargs astropy
`

Reply:
Ah, I think I see what's confusing here:
Image generation is not 100% platform portable, but at the same time we cannot tolerate any level of difference between reference images and the ones we get from CI. As a consequence, we can't run image tests locally. I think this is why you're seeing differences on your side, but CI is the only authority in this context.

Reply:
oh! I think I get it 
Consider using a Docker container that mimics the CI environment for local development. This can help reduce discrepancies between local runs and CI, though it may not completely eliminate differences in image generation.

Reply:
@astrofrog and @Cadair set this up, so maybe they can comment from the infrastructure side.